packages/core/src/stepFunctions/activation.ts

```diff
@@ -124,6 +126,19 @@ export function initalizeWebviewPaths(
     }
 }
 
+async function startLspServer(extensionContext: vscode.ExtensionContext) {
+    const perflog = new PerfLog('stepFunctions: start LSP client/server')
+    await ASLLanguageClient.create(extensionContext)
+    perflog.done()
+}
+
+function isASLFileOpen() {
+    return some(
```

Comment:
I think we're trying? to remove lodash as a dependency. Are you able to do this without using it?

Reply:
I misunderstood, I believe I need to use this `some()`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some. Will update.